app/lib/features/profile/pages/my_profile_page.dart

```diff
@@ -185,7 +167,8 @@ class MyProfile extends ConsumerWidget {
                   ),
                 ),
                 Container(
-                  width: MediaQuery.of(context).size.width - 100,
+                  width: isDesktop ? 400 : MediaQuery.of(context).size.width -15,
```

Comment:
Okay, I think we need to clarify _when_ `isDesktop` is okay to be used... because here it seems out of place again. A local width should not be informed by the platform but by the available space. The latest iPad in Landscape-Mode has a higher resolution than a MacBook Air 2017 - yet the later would be understood as a `isDesktop` while the former is not. And in neither case can you be sure you are on full-screen view (I almost never have Acter on full-screen). We need to think in a responsive pattern rather than this black-and-white pattern.

If you have a default size it should be that is meant to be responsive (like the MediaQuery used here - very good!) but it must tap-out for certain sizes, please use [`BoxConstraints`](https://api.flutter.dev/flutter/rendering/BoxConstraints-class.html):

```suggestion
                  constraints: const BoxConstraints(maxWidth: 400, width: MediaQuery.of(context).size.width -15),
```